internal/controller/streams.go

```diff
@@ -421,18 +423,26 @@ func ConnectStreams(ctx context.Context, logger *zap.SugaredLogger, envMap Envir
 	// Falco channels communicate news events between http server and our network flows strea,
 	falcoEventChan := make(chan string)
 	http.HandleFunc("/", NewFalcoEventHandler(falcoEventChan))
-	falcoEvent := &http.Server{Addr: falcoPort}
 	// Start our falco server and have it passively listen, if it fails, try to just restart it.
 	go func() {
 		for {
-			err := falcoEvent.ListenAndServe()
+			// Create a custom listener, this listener has SO_REUSEADDR option set by default
+			listener, err := net.Listen("tcp", falcoPort)
```

Comment:
https://cs.opensource.google/go/go/+/master:src/net/sockopt_aix.go;l=26?q=SO_REUSEADDR&ss=go%2Fgo:src%2Fnet%2F

Seems like this is now a default option